### DISCUSSION TITLE
新圣遗物的图片尺寸看上去好大，是我的错觉还是它真的很大

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#432] [General]
![image](https://user-images.githubusercontent.com/9006264/143199952-1318284d-a503-4811-b7fe-255da74d9bc9.png)
![image](https://user-images.githubusercontent.com/9006264/143199971-ba394c08-718e-47ca-bc3f-fc333804dfdb.png)

**Comment by Arondight** [comment]
是它真的好大
